### PR TITLE
ToolbarsExplorer 1.7 and 20230217-dev

### PR DESCRIPTION
**Pull request by ABuffEr**
### Release information

- Name: toolbarsExplorer
- Author: Alberto Buffolino
- Repo: https://github.com/ABuffEr/toolbarsExplorer
- Version: 1.7, 20230217-dev
- Update channel: stable, dev
- NVDA compatibility: 2017.3 and beyond

### Changelog

* updated translations;
* updated code and compatibility flag.

### Additional information

The two packages are substantially the same (code, I mean).

### CODE REVIEW

**Comment by nvdaes** [comment]
@ABuffEr Can you fix conflicts after merging master? Cheers.

**Comment by ABuffEr** [comment]
Hi @nvdaes ,
apparently, I resolved on my fork, but I have no write access here. Maybe should I make a new PR?

**Comment by nvdaes** [comment]
Conflicts are fixed now here. I'll review the add-on manifest (as required) in the evening.

**Review by nvdaes — APPROVED** [review]

**Pull request merged**
